scripts/check_format_cpp.sh

```diff
@@ -64,7 +64,9 @@ if [ $n_unformated_files -ne 0 ]; then
         echo "    $file"
     done
     if [ $apply_patch -eq 1 ]; then
-        $clang_format_executable -i $file
+        for file in $cpp_source_files; do
```

Comment:
Can't you simply move the `if` clause inside `for file in ${unformatted_files[@]}`?

Reply:
Done